experiments/boolean_logic/evolve_NN.py

```diff
@@ -56,23 +55,29 @@
 
 for i in range(cf.generations):
     start = time.time()
+   
     for j in range(cf.genomes):
         # Set the DAC voltages
-        for k in range(cf.genes-1):
-            controlVoltages[k] = genePool.MapGenes(
-                                    cf.generange[k], genePool.pool[j, k])
-
-        # Set the input scaling
-        x_scaled = x * genePool.MapGenes(cf.generange[-1], genePool.pool[j, -1])
-
+#        for k in range(cf.genes-1):
+#            controlVoltages[k] = genePool.MapGenes(
+#                                    cf.generange[k], genePool.pool[j, k])
+        #map input 
+        x_mapped = cf.inputrange[0] + x * ( cf.inputrange[1] -  cf.inputrange[0])
```

Comment:
Is this specific of this model or why is this done? 
Can you also please add your implementation for choosing the inputs? 
I would like this to be a good version so that we can all work on this and include the relevant parts to the new version, so please include only changes that are general and not specific. If there are specific changes that you must do for the script to work properly, please make a comment about it.
I need your help on making this merge as efficiently as possible and I would like to use your changes on my search next week so we don work double. I would really appreciate if you can help me with this.